testsuite/tests/typing-safe-linking/Makefile

```diff
@@ -3,6 +3,13 @@
 SOURCES = a.ml b_bad.ml
 OBJECTS = $(SOURCES:%.ml=%.cmo)
 
+default:
+	@if $(SAFE_STRING) ; then \
+	  echo " ... testing 'b_bad.ml' => skipped" ; \
```

Comment:
@garrigue - I think it's correct that since `-unsafe-string` is unavailable when OCaml is configured with `-safe-string` that this test should be completely skipped, as it's not possible to build `a.cmo` with mutable strings?